common/build.gradle

```diff
@@ -89,6 +89,7 @@ dependencies {
   api group: 'org.springframework.security',        name: 'spring-security-jwt',              version: '1.0.8.RELEASE'
   api group: 'org.springframework.security',        name: 'spring-security-core',             version: springSecurityVersion
   api group: 'org.springframework.security',        name: 'spring-security-config',           version: springSecurityVersion
+  api group: 'com.cloudera.crypto',                 name: 'openssl-provider',                 version: clouderaCryptoOpenSslProvider
```

Comment:
Sanity checks: Is it safe to have this on the classpath for non-FIPS deployments as well?

Reply:
Yes, it is safe as it needs to be configured through the `java.security` conf file.